src/main/java/servlets/SignInServlet.java

```diff
@@ -10,38 +9,66 @@
 import javax.servlet.http.HttpSession;
 import java.io.IOException;
 
-@WebServlet(name = "SignIn", value = "/signin")
+
+@WebServlet(name = "SignIn", value = "/sign_in")
 public class SignInServlet extends HttpServlet {
-	
+
 	private String checker;
-	
-	@Override protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+
+	@Override
+	protected void doGet(HttpServletRequest req, HttpServletResponse resp)
 			throws ServletException, IOException {
-		doPost(req, resp);
+		HttpSession session = req.getSession();
+		String login;
+		String password;
+		login = (String) session.getAttribute("login");
+		password = (String) session.getAttribute("password");
+		if (login != null && password != null) {
+			if (login.equalsIgnoreCase("admin")) {
+				req.getRequestDispatcher("/adminpage.jsp").forward(req, resp);
+			} else {
+				req.getRequestDispatcher("/pricelist.jsp").forward(req, resp);
+			}
+		} else {
+			req.getRequestDispatcher("/signin.jsp").forward(req, resp);
+		}
 	}
-	
-	@Override protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+
+	@Override
+	protected void doPost(HttpServletRequest req, HttpServletResponse resp)
 			throws ServletException, IOException {
-//		req.getRequestDispatcher("/signin")
-//				.forward(req, resp);
 		HttpSession session = req.getSession();
-		String      login;
-		String      password;
-		if (session != null) {
-			login = (String) session.getAttribute("login");
-			password = (String) session.getAttribute("password");
-			checker = checkUser(login, password);
-			
-			if (checker.equalsIgnoreCase("ban")) {
-				req.getRequestDispatcher("/banneduser.jsp")
-						.forward(req, resp);
-			}
+		String login;
+		String password;
+		login = (String) req.getParameter("login");
+		password = (String) req.getParameter("password");
+		//TODO: logic of verification login and password in DB
+		//TODO: logic of checking login's status
+		checker = "active";
+		if (login.equals("admin")) {
+			checker = "admin";
+		}
+		if (login.equals("banned")) {
+			checker = "banned";
+		}
```

Comment:
Keep it more simple, like this:
checker = (condition) ? "admin" : (condition) ? "banned" : active;